pkgs/development/python-modules/sv-ttk/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib
+, pkgs
+, python3
+, fetchPypi
+}:
+python3.pkgs.buildPythonPackage rec {
+  pname = "sv-ttk";
+  version = "2.4.5";
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "sv_ttk";
+    hash = "sha256-ysRhRxrml+wmluH8F5AE7vZYXrTNUg5ZzI+26jwpOpc=";
+  };
+
+  meta = with lib; {
+    description = "A gorgeous theme for Tkinter/ttk, based on the Sun Valley visual style";
+    homepage = "https://github.com/rdbende/Sun-Valley-ttk-theme";
+    license = licenses.mit;
+    platforms = platforms.linux;
```

Comment:
See https://github.com/rdbende/Sun-Valley-ttk-theme/blob/main/setup.py#LL45C1-L45C46

```suggestion
```

Reply:
Whoops - that went a bit fast. Good catch :)